Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -148,6 +148,9 @@ static bool shouldAllowNonPointerCursorForElement(const Element& element)
         return true;
 #endif
 
+    if (is<HTMLTextFormControlElement>(element))
+        return !element.document().userActionElements().isFocused(element);
```

Comment:
```suggestion
        return !element.focused();
```

Reply:
thanks! got confused by the private `isUserActionElementFocused`